exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -128,55 +232,202 @@ void createTestKitWithoutServices() {
     assertThrows(exceptionType, testKitBuilder::build);
   }
 
-  public static final class TestServiceModule extends AbstractServiceModule {
+  @Test
+  void createEmptyBlock() {
+    try (TestKit testKit = TestKit.forService(TestServiceModule.class)) {
+      Block block = testKit.createBlock();
+      assertThat(block.getNumTransactions()).isEqualTo(0);
 
-    private static final TransactionConverter THROWING_TX_CONVERTER = (tx) -> {
-      throw new IllegalStateException("No transactions in this service: " + tx);
-    };
+      testKit.withSnapshot((view) -> {
+        Blockchain blockchain = Blockchain.newInstance(view);
+        assertThat(blockchain.getHeight()).isEqualTo(1);
+        assertThat(block).isEqualTo(blockchain.getBlock(1));
+        return null;
+      });
+    }
+  }
 
-    @Override
-    protected void configure() {
-      bind(Service.class).to(TestService.class).in(Singleton.class);
-      bind(TransactionConverter.class).toInstance(THROWING_TX_CONVERTER);
+  @Test
+  void afterCommitSubmitsTransaction() {
+    Block nextBlock;
```

Comment:
(Here and elsewhere) Why don't we declare these variables inside try-catch and initialize immediately where they are needed?